### ISSUE TITLE
Add new Beef Icon

### ISSUE THREAD

**Issue by hakrishi** [walkxcode/dashboard-icons#302]
### Type

Icon request

### Info

Ref:
https://github.com/beefproject/beef
https://beefproject.com/

### Icon(s)

![image](https://user-images.githubusercontent.com/59293085/222956133-5dbe0c42-c6ba-4d0e-be2a-b5c4035983ff.png)

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/beef.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/beef-light.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/svg/beef.svg

**Comment by hakrishi** [comment]
Thanks for this @walkxcode!! Just curious which tool you use for easily creating/converting the images?

**Comment by walkxcode** [comment]
I used https://deeditor.com to change the colors. And https://ezgif.com/svg-to-png to convert from SVG to PNG. (Deeditor also has a conversion option available, but I avoid using it as it gives massive file sizes compared to ezgif).

**Comment by hakrishi** [comment]
thanks, which tool do you use to resize the image?

Going forward I'll start creating PRs and submitting new icons 🙂
